### PR TITLE
improve: use errors.New to replace fmt.Errorf with no parameters

### PR DESCRIPTION
**Pull request by ChengenH**
use errors.New to replace fmt.Errorf with no parameters

### CODE REVIEW

File: op-e2e/external_geth/main.go [hunk 1]
```diff
@@ -46,10 +46,10 @@ func run(configPath string) error {
 
 	binPath, err := filepath.Abs("op-geth")
 	if err != nil {
-		return fmt.Errorf("could not get absolute path of op-geth")
+		return errors.New("could not get absolute path of op-geth")
```
Comment (ChengenH) [inline op-e2e/external_geth/main.go:49]: It should be the same as `op-e2e/external_geth/main.go` line 90, use errors.New to replace fmt.Errorf with no parameters.

File: op-e2e/external_geth/main.go [hunk 2]
```diff
@@ -66,7 +66,7 @@ func run(configPath string) error {
 
 	fmt.Printf("==================    op-geth shim encoding ready-file   ==========================\n")
 	if err := external.AtomicEncode(config.EndpointsReadyPath, sess.endpoints); err != nil {
-		return fmt.Errorf("could not encode endpoints")
+		return errors.New("could not encode endpoints")
```
Comment (ChengenH) [inline op-e2e/external_geth/main.go:69]: ditto

File: op-e2e/external_geth/main.go [hunk 3]
```diff
@@ -164,7 +164,7 @@ func execute(binPath string, config external.Config) (*gethSession, error) {
 	for enginePort == 0 || httpPort == 0 {
 		match, err := matcher.Match(sess.Err)
 		if err != nil {
-			return nil, fmt.Errorf("could not execute matcher")
+			return nil, errors.New("could not execute matcher")
```
Comment (ChengenH) [inline op-e2e/external_geth/main.go:167]: ditto

File: op-e2e/external_geth/main.go [hunk 4]
```diff
@@ -101,7 +101,7 @@ func awaitExit(sess *gexec.Session) error {
 		case <-sess.Exited:
 			return nil
 		case <-time.After(30 * time.Second):
-			return fmt.Errorf("exiting after 30 second timeout")
+			return errors.New("exiting after 30 second timeout")
```
Comment (ChengenH) [inline op-e2e/external_geth/main.go:104]: ditto

**Comment by ajsutton** [comment]
Thanks bug I'm going to close this. We've previously resolved that we're ok with using `fmt.Errorf` in this case - there's plenty of other cases across the codebase and it's not worth spending time enforcing as both forms work just as well.

**Pull request closed without merging**